### PR TITLE
superfluous and missing include

### PR DESCRIPTION
**Pull request by mxmlnkn**

### CODE REVIEW

**Comment by ax3l** [comment]
looks all right, thank you!

File: src/libPMacc/include/nvidia/functors/Assign.hpp [hunk 1]
```diff
@@ -22,8 +22,6 @@
 
 #pragma once
 
-#include "pmacc_types.hpp"
```
Comment (BenjaminW3) [inline src/libPMacc/include/nvidia/functors/Assign.hpp]: The `HDINLINE` used in this file is defined in `pmacc_types.hpp` so I think this is not superfluous.

Comment (ax3l) [inline src/libPMacc/include/nvidia/functors/Assign.hpp]: oh dang you are right!

Comment (ax3l) [inline src/libPMacc/include/nvidia/functors/Assign.hpp]: arrr, I must have been sleeping.... :S

Comment (mxmlnkn) [inline src/libPMacc/include/nvidia/functors/Assign.hpp]: Sorry for that, I didn't know that.
At least the review process works in the end.

Comment (ax3l) [inline src/libPMacc/include/nvidia/functors/Assign.hpp]: no prob! :)

**Pull request merged**